src/SDKs/_metadata/recoveryservicessiterecovery_resource-manager.txt

```diff
@@ -1,17 +1 @@
-﻿Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/recoveryservicessiterecovery/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\Github\azure-sdk-for-net\tools\..\src\SDKs\
-2018-05-08 13:44:39 UTC
-1) azure-rest-api-specs repository information
-GitHub fork: Azure
-Branch:      master
-Commit:      f0face31d69f4a7191f9db6143c0a1a01f398d89
```

Comment:
@sriramvu Please run msbuild build.proj and regenerate the code again

Reply:
regenerated code and SdkInfo_SiteRecoveryManagementClient.cs got some changes.